src/avfoundation.cs

```diff
@@ -12979,7 +13468,7 @@ interface AVSpeechSynthesizerDelegate {
 		void WillSpeakRangeOfSpeechString (AVSpeechSynthesizer synthesizer, NSRange characterRange, AVSpeechUtterance utterance);
 	}
 
-	[NoWatch, NoTV, NoMac, iOS (11,0)]
+	[NoWatch, NoTV, Mac (12,0), iOS (11,0)]
```

Comment:
catalyst ?

Reply:
No need, it is the min version of catalyst.